libs/components/a11y/src/lib/modules/shared/sky-a11y-resources.module.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * NOTICE: DO NOT MODIFY THIS FILE!
  * The contents of this file were automatically generated by
- * the 'ng generate @skyux/i18n:lib-resources-module modules/shared/sky-a11y' schematic.
+ * the 'ng generate @skyux/i18n:lib-resources-module lib/modules/shared/sky-a11y' schematic.
```

Comment:
☝️ This comment is auto-generated by the `@skyux/i18n` schematic, but is a bit useless to us since `ng g @skyux/i18n:lib-resources-module` can't be executed within the context of the monorepo without building the library and `npm link`ing it.